### PR TITLE
DEVDOCS-2487: Create Deprecations and Sunsets page

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2487](https://jira.bigcommerce.com/browse/DEVDOCS-2487)

## What changed?
* added a stand alone Deprecations page

### CODE REVIEW

File: docs/api-docs/getting-started/deprecations-and-sunsets.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### On this page
+- [Deprecations](#deprecations)
+- [Sunsets](#sunsets)
+- [Related resources](#related-resources)
+
+</div>
+
+## Deprecations
+
+The following endpoints are deprecated. We discourage using these endpoints as they are no longer supported by BigCommerce. Instead, consider using the provided V3 replacements.
```
Comment (traci707) [inline docs/api-docs/getting-started/deprecations-and-sunsets.md:14]: The following endpoints are deprecated. We discourage using these endpoints as BigCommerce no longer supports them. Instead, consider using the provided V3 replacements.

File: docs/api-docs/getting-started/deprecations-and-sunsets.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+> * In V3, `options` and `modifiers` attach directly to products. Use `options` and `modifiers` together to access the contents of the former V2 `options` response.
+> * `option_sets` endpoint is intentionally not available in the V3 API. See [Difference between V2 and V3 Catalog APIs](https://developer.bigcommerce.com/api-docs/store-management/catalog/v2-vs-v3#difference-between-v2-and-v3-catalog-apis) for more information.
+
+</div>
+</div>
+</div>
+
+## Sunsets
+
+The following operations are scheduled to be removed.
```
Comment (traci707) [inline docs/api-docs/getting-started/deprecations-and-sunsets.md:41]: BigCommerce has scheduled the removal of the following operations.

File: docs/api-docs/getting-started/deprecations-and-sunsets.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+| Operation | Endpoints | Sunset | Replacement |
+|-|-|-|-|
+| `DELETE Collection` | `/v2/customers`| March 31, 2021| [`DELETE /v3/customers`](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api/customers/customersdelete)|
+| `DELETE Collection` | `/v2/option_sets`| May 10, 2020| None; can still be deleted individually by ID.|
+| `DELETE Collection` | `/v2/products`| May 10, 2020| [`DELETE /v3/catalog/products`](https://developer.bigcommerce.com/api-reference/catalog/catalog-api/products/deleteproducts)|
+
+The following properties are scheduled to be removed.
```
Comment (traci707) [inline docs/api-docs/getting-started/deprecations-and-sunsets.md:49]: BigCommerce has scheduled the removal of the following properties.

**Review by traci707 — CHANGES_REQUESTED** [review]

**Review by traci707 — APPROVED** [review]
LGTM

File: docs/api-docs/apps/guide/apps-05-oauth.md [hunk 4]
```diff
@@ -87,8 +87,8 @@ GET /auth?code=qr6h3thvbvag2ffq&scope=store_v2_orders&context=stores/g5cd38 HTTP
 > ### Note
 >
 > * When your app receives a new token, any previously issued token is invalidated.
-> * As a best practice, your app should validate this list to ensure that it matches your app's needs, and fail if it does not. At this time, the user can not pick and choose scopes. The dialog presented to the user requires the user to approve all scopes or none.
-> * The request comes from the client browser, rather than directly from BigCommerce. This allows you to use a non-publicly available auth callback URL while testing your app.
+> * As a best practice, your app should validate the list of scopes to ensure that it matches your app's needs and fail if it does not. At this time, the user can not pick and choose scopes. The dialog presented to the user requires the user to approve all scopes or none. For more information about available scopes, see [OAuth scopes](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes).
```
Comment (uwikaiddi) [inline docs/api-docs/apps/guide/apps-05-oauth.md:90]:
```suggestion
> * As a best practice, your app should validate the list of scopes to ensure that it matches your app's needs and fails if it does not. At this time, the user cannot pick and choose scopes. The dialog presented to the user requires the user to approve all scopes or none. For more information about available scopes, see [OAuth scopes](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes).

```

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**